content/applications/hr/payroll.rst

```diff
@@ -185,13 +195,13 @@ useful to add may be a `Bonus`.
 To view all the various structures for each structure type, go to :menuselection:`Payroll -->
 Configuration --> Structures`.
 
+Each :ref:`structure type <payroll/structure-types>` lists the various structures associated with
+it. Each structure contains a set of rules that define it.
+
```

Comment:
I thought this section read well, and I believe the images could be removed and it would still make sense (that's how I test in my mind if it's explained enough or not). I did keep it simple, but I think concise.